backend/fms_core/viewsets/sample_next_step.py

```diff
@@ -170,6 +170,12 @@ def labwork_info(self, request, *args, **kwargs):
             if protocol.id not in sample_next_step_by_protocol.keys():
 
                 protocolSteps = Step.objects.filter(protocol=protocol)
+
+                # Make sure that at least one workflow uses one of the steps for this protocol, since
+                # labwork doesn't need protocols that are not used by any workflow (Infinium...)
+                workflows = Workflow.objects.filter(steps__in=protocolSteps)
```

Comment:
`if Workflow.objects.filter(steps__in=protocolSteps).exists():` should be lighter and clearer.

Reply:
Excellent - fixed. 👍